File: javascript/polymorphism.js
class calc{
    add(){
        console.log("inside no arg method");
    }
    add(num){
        console.log("inside one arg method");
    }
    add(num1,num2){
        console.log("inside two arg method");
    }
}
var cal=new calc()
cal.add(10,20);

File: objectoriented prog/polymorphism.py
#more than one form
#1) method overloading
class Maths:
    def add(self):
        print("inside no arg")

    def add(self,num1):
        print("inside one arg")

    def add(self,num1,num2):
        print("inside two arg")

m=Maths()
m.add(10,20)